src/plugins/kibana/public/dashboard/index.js

```diff
@@ -68,6 +68,7 @@ app.directive('dashboardApp', function (Notifier, courier, AppState, timefilter,
       if (dash.timeRestore && dash.timeTo && dash.timeFrom && !getAppState.previouslyStored()) {
         timefilter.time.to = dash.timeTo;
         timefilter.time.from = dash.timeFrom;
+        timefilter.refreshInterval = dash.refreshInterval;
```

Comment:
I think we need a conditional around this line, otherwise loading up an old dashboard logs the following error in the console because dash.refreshInterval is undefined:

![screen shot 2016-06-08 at 5 08 17 pm](https://cloud.githubusercontent.com/assets/6239176/15911013/c8c65e48-2d9b-11e6-94c4-fdd010212b74.png)